buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -179,6 +179,11 @@ private object Libraries {
 
     const val LAUNCHPAD_COMPOSE = "com.github.BottleRocketStudios:Android-LaunchPad-Compose:0.2.0"
 
+
+    // Accompanist
```

Comment:
Nice to have a quick, (cmd) clickable link near the dependency:

```suggestion
    // Accompanist
    // https://github.com/google/accompanist/releases
```